mne_bids/dig.py

```diff
@@ -34,30 +34,8 @@ def _handle_electrodes_reading(electrodes_fname, coord_frame,
     logger.info('Reading electrode '
                 'coords from {}.'.format(electrodes_fname))
     electrodes_dict = _from_tsv(electrodes_fname)
-    # First, make sure that ordering of names in channels.tsv matches the
-    # ordering of names in the raw data. The "name" column is mandatory in BIDS
-    ch_names_raw = list(raw.ch_names)
     ch_names_tsv = electrodes_dict['name']
 
-    if ch_names_raw != ch_names_tsv:
-        msg = ('Channels do not correspond between raw data and the '
-               'channels.tsv file. For MNE-BIDS, the channel names in the '
-               'tsv MUST be equal and in the same order as the channels in '
-               'the raw data.\n\n'
-               '{} channels in tsv file: "{}"\n\n --> {}\n\n'
-               '{} channels in raw file: "{}"\n\n --> {}\n\n'
-               .format(len(ch_names_tsv), electrodes_fname, ch_names_tsv,
-                       len(ch_names_raw), raw.filenames, ch_names_raw)
-               )
-
-        # XXX: this could be due to MNE inserting a 'STI 014' channel as the
-        # last channel: In that case, we can work. --> Can be removed soon,
-        # because MNE will stop the synthesis of stim channels in the near
-        # future
-        if not (ch_names_raw[-1] == 'STI 014' and
-                ch_names_raw[:-1] == ch_names_tsv):
-            raise RuntimeError(msg)
-
```

Comment:
why did you remove this? did we have an issue with that somewhere?

I mean, it's RECOMMENDED in BIDS ... but we could still raise a warning if someone doesn't comply?

Reply:
I removed a similar test in #698, so if we change this one here to a warning, we should revert the relevant part to #698 and change it into a warning too

Reply:
I'm okay with no warning here if no warning in #698 . 

Lmk what you guys decide?

Reply:
okay, fine to remove it then